dataload/import_to_elasticsearch.py

```diff
@@ -243,7 +249,29 @@ def maybe_create_index(index_name=ES_INDEX):
                             }
                         }
                     },
-
+                    "GNCanonicalRecipientOrgId": {
```

Comment:
Is this elasticsearch document structure defined by any other schema, like with the CSV downloads, or is this purely internal to grantnav, and it can be whatever we choose in the same vein Django model?

Reply:
Yeah this is purely internal to grantnav, we choose what data it holds. The main difference from a django is that there are many more fields which aren't mentioned here which are created automatically/inferred from the data.